docs/content/direct/multi-wec-aggregated-status.md

```diff
@@ -0,0 +1,197 @@
+
+Both `BindingPolicy` and `Binding` inherit the `DownsyncModulation` extension where this flag resides.
+
+---
+
+### 3. Status Aggregation Rules
+
+The aggregation logic depends on the **workload object kind**.
+
+#### Case 1 – Known Workload Kinds
+
+For workload kinds recognized by ArgoCD, the controller applies predetermined field aggregation rules consistent with ArgoCD’s native health evaluation. These include:
+
+##### **Deployment**  
+  - `replicas`, `updatedReplicas`, `availableReplicas`, `readyReplicas`: use `min()` across clusters to reflect the least ready state.  
+  - `conditions`: apply three-value logic per condition type:  
+    - `False` if any cluster reports `False`.  
+    - `True` if all clusters report `True`.  
+    - Otherwise, `Unknown`.  
+  - `observedGeneration`: use the minimum observed generation.  
+  - `unavailableReplicas`: use `max()` to represent the worst state.
+
+##### **StatefulSet**  
+  - `replicas`, `readyReplicas`, `currentReplicas`, `updatedReplicas`: use `min()`.  
+  - `conditions`: three-value logic as above.
+
+##### **DaemonSet**  
+  - `currentNumberScheduled`, `desiredNumberScheduled`, `numberAvailable`, `numberReady`: use `min()`.  
+  - `conditions`: three-value logic.
+
+##### **ReplicaSet**  
+  - `replicas`, `readyReplicas`, `availableReplicas`: use `min()`.  
+  - `conditions`: three-value logic.
+
+##### **Job**  
+  - `active`, `succeeded`, `failed`: use `min()` for all numeric fields to represent the least-ready state across clusters.  
+  - `conditions`: three-value logic.
+
+##### **CronJob**  
+  - `lastScheduleTime`: use the latest timestamp across clusters.  
+  - `conditions`: three-value logic.
+
+##### **Pod**  
+  - `phase`: aggregate by prioritizing `Failed` > `Unknown` > `Running` > `Succeeded` > `Pending`.  
+  - `conditions`: three-value logic.
+
+**Three-value logic for Conditions:**  
+- `False` if any cluster reports `False` for a condition type.  
+- `True` if all clusters report `True`.  
+- Otherwise, `Unknown`.
+
+**Timestamps:**  
+- For all condition timestamps (`lastTransitionTime`), take the most recent timestamp across clusters.
+
+**Strings (Reason, Message):**  
+- For each condition, take the `Reason` and `Message` from the condition with the most recent `lastTransitionTime`.
+
+- **Slices and maps:**  
+  - Match items by keys (e.g., condition `type`) before per-field aggregation.
+
+#### Case 2 – Unknown Workload Kinds
+
+For workload kinds not recognized by ArgoCD, the controller performs generic per-field aggregation using the same logical rules as above but constrained for general applicability:
+
+- **Numeric fields:** use `min()` to represent the least-ready state across clusters.  
+- **Conditions:** apply the same three-value logic (False if any False, True if all True, otherwise Unknown).  
+- **Timestamps:** take the latest `lastTransitionTime`.  
+- **Strings:** pick the most recent `Reason` and `Message` values.  
```

Comment:
Again, these are part of the Condition aggregation. Do not state them as peers to Conditions. Group the truth value, timestamp, reason, and message handling together in one Condition section.